fields.f90

```diff
@@ -1561,6 +1561,11 @@ subroutine FLReadInput(NPTorder,Npolyads,Natoms,Nmodes,Jrot)
             !
          endif 
          !
+         if (.not. symmetry_defined) then
```

Comment:
@Trovemaster Sergey, this is my solution to the diagonalizer-symmetry ordering issue. The if statement below was the one causing the segfault; `job%nroots` was the array being accessed without it first being allocated in the code handling the symmetry input block.

Reply:
Thanks! This makes sense.